src/main/java/com/team4/testingsystem/controllers/TestsController.java

```diff
@@ -40,6 +40,11 @@ public Test getById(@PathVariable("id") long id) {
         return testsService.getById(id);
     }
 
+    @ApiOperation(value = "Get all tests assigned to the current user")
+    @GetMapping(path = "/assigned")
```

Comment:
We have an operation 'assigned test', so '/assigned' mapping for tests that actually returns all the tests of the user seems a bit confusing. Let's have another mapping for it.

BTW, do we really need to getAll operation here? If not, I think '/' mapping could be fine here

Reply:
Yeah, I was thinking about replacing `GET /` mapping, moreover it's marked as experimental